### ISSUE TITLE
Lab1, Βήμα 13Γ

### ISSUE THREAD

**Issue by Yenn1s** [slp-ntua/patrec-labs#2]
Καλησπέρα, 

Δεν είμαι σίγουρος τι πρέπει να σχεδιαστεί στην καμπύλη εκμάθησης του ταξινομητή. 
Θα πρέπει να κάνουμε σταδιακά fit αυξάνοντας το training set και να αποτυπώσουμε το Accuracy στην πορεία αυτή;

**Comment by georgepar** [comment]
Όπως σωστά λες, το learning curve είναι η απεικόνιση μιας μετρικής (πχ accuracy) ως προς τον αριθμό των training examples.

Μπορείτε να υλοποιήσετε μόνοι σας αυτή τη διαδικασία που κάνετε train μοντέλα με αυξανόμενο αριθμό παραδειγμάτων, ή να χρησιμοποιείσετε (και να καταλάβετε τον κώδικα) της έτοιμης συνάρτησης που παρέχει το sklearn (https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.learning_curve.html)
